libs/perun/openapi/src/lib/api.module.ts

```diff
@@ -49,7 +49,7 @@ import { VosManagerService } from './api/vosManager.service';
     VosManagerService ]
 })
 export class ApiModule {
-    public static forRoot(configurationFactory: () => Configuration): ModuleWithProviders<any> {
+    public static forRoot(configurationFactory: () => Configuration): ModuleWithProviders {
```

Comment:
ModuleWithProviders requires one type argument. You can use `<any>` or according to official angular guide `<ApiModule>` in this case.

Reply:
Fixed, for some reason this argument gets removed every time openAPI is generated.